mkdocs.yml

```diff
@@ -132,19 +132,29 @@ nav:
       - limitador-operator/doc/logging.md
   - 'Multicluster Gateway Controller':
     - 'Overview': multicluster-gateway-controller/README.md
+    - 'Architecture': multicluster-gateway-controller/docs/architectural-overview.md
```

Comment:
this doc is in the architecture repo. Is it possible to use that from there?

Reply:
Ah! Wondered where it was (that makes sense). Yep, I should be able to pull that in as well...